libshvbroker/src/brokerapp.cpp

```diff
@@ -240,6 +246,30 @@ void do_open_function(const auto& ldap_lib, auto fn_name, auto& fn_field)
 }
 #endif
 
+std::optional<AzureConfig> BrokerApp::azureConfig()
+{
+	return m_azureConfig;
+}
+
+namespace {
+auto transform_cli_group_mapping(auto cli_group_mapping)
+{
+	std::vector<GroupMapping> group_mapping;
+	std::transform(cli_group_mapping.begin(),
+				   cli_group_mapping.end(),
+				   std::back_inserter(group_mapping),
+				   [] (const auto& mapping) {
+					   return GroupMapping {
+						   .nativeGroup = mapping.asList().at(0).asString(),
```

Comment:
Is this intentional to crash app in case of invalid config?

Reply:
Fixed. Good catch.